print_format/sales_invoice.jinja

```diff
@@ -80,15 +80,25 @@
             {% endif %}
 
             {% if doc.from_date and doc.to_date %}
+                {% set from_date = doc.from_date %}
+                {% set to_date = doc.to_date %}
+            {% endif %}
+            {% set timesheet = frappe.get_doc("Timesheet", doc.timesheets.time_sheet)%}
+            {% if timesheet.start_date and timesheet.end_date %}
+                {% set from_date = timesheet.start_date %}
+                {% set to_date = timesheet.end_date %}
+            {% endif %}
```

Comment:
Ich denke das sollte nicht Aufgabe des Druckformats sein. Abgesehen davon verwenden wir beispielsweise eine andere Logik (es wird immer ein ganzer Monat vom ersten bis letzten Tag abgerechnet). Besser ein Server Script erstellen, das die entsprechenden Werte nach Belieben in der Rechnung setzt.